vault/resource_identity_entity_alias_test.go

```diff
@@ -101,11 +104,17 @@ func testAccIdentityEntityAliasConfig(entityName string, dupeAlias bool, altTarg
 resource "vault_identity_entity" "entityA" {
   name = "%s-A"
   policies = ["test"]
+  metadata = {
```

Comment:
We should probably update the tests for this newly added field.

Is this addition required for this fix?

Reply:
Added to its own test